test/errors/implicitModule.compopts

```diff
@@ -0,0 +1,2 @@
+
+--brief-errors
```

Comment:
As a style note that I wouldn't bother changing in this PR, but maybe for future reference.  My sense is that even though we support the `.1.good`, `.2.good` capability, the preferred style is probably to use named .good files for non-default behavior like `implicitSubModule-brief.good`.   This has the benefits of making the filename more descriptive as to the contents and of not getting into memory issues like "Does this use 0- or 1-based numbering?"  The fact that so many of us didn't even realize that there was the ability to number the .good files also suggests to me that most people names them.

So I'd have used:
```

--brief-errors   #implicitModule-brief.good
```

for this compopts.  Or if/when brief became the default, I'd have used:

```

--long-errors  # implicitModule-long.good
```